package.json

```diff
@@ -38,7 +38,7 @@
     "clean": "yarn workspaces run clean",
     "lint": "yarn workspaces run lint",
     "lint:fix": "yarn workspaces run lint:fix",
-    "changeset:version:custom": "yarn changeset version && yarn workspace @sphinx-labs/contracts run generate"
+    "changeset:version:custom": "yarn changeset version && yarn workspace @sphinx-labs/contracts run write-version"
```

Comment:
I had to make some changes to the release process to handle using the commit hash instead of the version number. The main difference is that we probably shouldn't just regenerate the versions in the standard build command b/c then we'd be frequently switching the hash out for different ones that may not be valid at all.

So instead, I've updated the build process to only run the `write-version` script when we're actually doing a release.